db/migrate/20220310080950_create_users.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :email
```

Comment:
FYI
実際のサービスでは以下のようにバリデーションを付与してあげるのが一般的です。
```suggestion
      t.string :email, null: false
```